ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/_cq_dialog/.content.xml

```diff
@@ -38,14 +38,27 @@
                             active="{Boolean}true"
                             selected="{Boolean}true">
                         <items jcr:primaryType="nt:unstructured">
-                            <default
+                            <columns
                                     jcr:primaryType="nt:unstructured"
-                                    sling:orderBefore="guideGlobalLazyField"
-                                    sling:resourceType="granite/ui/components/coral/foundation/form/textfield"
-                                    fieldDescription="Please enter the date in the required format  &quot;yyyy-mm-dd&quot;."
-                                    fieldLabel="Default date"
-                                    emptyText="YYYY-MM-DD"
-                                    name="./default"/>
+                                    sling:resourceType="granite/ui/components/coral/foundation/fixedcolumns"
+                                    margin="{Boolean}true">
+                                <items jcr:primaryType="nt:unstructured">
+                                    <column
+                                            jcr:primaryType="nt:unstructured"
+                                            sling:resourceType="granite/ui/components/coral/foundation/container">
+                                        <items jcr:primaryType="nt:unstructured">
+                                            <default
+                                                    jcr:primaryType="nt:unstructured"
+                                                    sling:orderBefore="guideGlobalLazyField"
```

Comment:
not part of this PR: but we need to take care of this: remove sling:orderBefore="guideGlobalLazyField" as its not required here.